package.json

```diff
@@ -27,7 +27,6 @@
     "build-es5": "mkdir -p dist && babel src --out-dir dist --plugins=transform-es2015-modules-commonjs --source-maps inline",
     "build-es6": "mkdir -p dist-es6 && babel src --out-dir dist-es6 --source-maps inline",
     "clean": "rm -fr dist/* dist-es6/*",
-    "clean": "rm -fr dist/* dist-es6/*",
```

Comment:
Why remove the "clean" script? I'd prefer that we didn't. If there is a reason to do it, it should be done in a separate PR.